.github/workflows/main.yml

```
@@ -96,18 +97,40 @@ jobs:
       
-    - name: Build project
-      run: npm run build
+    - name: Performance benchmark
+      run: |
+        echo "📊 Running performance benchmark..."
+        START_TIME=$(date +%s%N)
+        npm run build
+        END_TIME=$(date +%s%N)
+        BUILD_TIME=$((($END_TIME - $START_TIME)/1000000))
+        echo "Build time: ${BUILD_TIME}ms"
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)